07/27/2024 05:31:13 PM Logging starts
Traceback (most recent call last):
  File "./popular_projects_snippets_dataset/flask/bodies/body_39_guide_9_8.py", line 28, in <module>
    raise ValueError("Cannot register a blueprint on itself")
ValueError: Cannot register a blueprint on itself
Command exited with non-zero status 1
1.60user 4.02system 0:00.83elapsed 672%CPU (0avgtext+0avgdata 104928maxresident)k
0inputs+24outputs (1major+23829minor)pagefaults 0swaps
